src/pk.c

```diff
@@ -6789,7 +6802,7 @@ WOLFSSL_BIGNUM* wolfSSL_DH_6144_prime(WOLFSSL_BIGNUM* bn)
  */
 WOLFSSL_BIGNUM* wolfSSL_DH_8192_prime(WOLFSSL_BIGNUM* bn)
 {
-    const char prm[] = {
+    static const char prm[] = {
```

Comment:
Why add `static`?

Reply:
the incumbent code was frivolously copying the `const` object to the stack before copying that copy to the caller-supplied `bn` using `wolfSSL_BN_hex2bn()`.  the copy to the stack triggered `gcc -Wframe-larger-than=2048`.

in round#1 I only fixed this one instance, but now that you remind me of it, I've added the `static` attribute to the rest of the `wolfSSL_DH_*_prime()` accessors, by the same rationale.